### PR TITLE
Update release links.

### PR DESCRIPTION
**Pull request by joerg84**
## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

File: 1.9/installing/cloud/aws/basic.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ nav_title: AWS EC2 Basic
 menu_order: 0
 ---
 
-You can create a DC/OS cluster for Amazon Web Services (AWS) using the <a href="https://downloads.dcos.io/dcos/stable/1.9.1/aws.html" target="_blank">DC/OS templates for AWS CloudFormation</a>.
+You can create a DC/OS cluster for Amazon Web Services (AWS) using the <a href="https://downloads.dcos.io/dcos/stable/1.9.2/aws.html" target="_blank">DC/OS templates for AWS CloudFormation</a>.
```
Comment (judithpatudith) [inline 1.9/installing/cloud/aws/basic.md]: Is there a reason that this is formatted as HTML while the other links look like markdown?

Comment (stbof) [inline 1.9/installing/cloud/aws/basic.md]: HTML formatted links will complicate the conversion to the new doc site. The link should be converted to Markdown syntax if possible.

Comment (joerg84) [inline 1.9/installing/cloud/aws/basic.md]: This is orthogonal to this PR....
I would need to change all links on the page for this to make sense... Let us rather do a cleanup PR which is focused on changing those links...
@judithpatudith @stbof
